src/main/java/com/iexec/common/replicate/ReplicateStatusCause.java

```diff
@@ -56,4 +58,16 @@
 
     ABORTED_BY_WORKER,
     CANNOT_REVEAL,
+    UNKNOWN;
+
+    public static ReplicateStatusCause getReplicateAbortCause(TaskAbortCause cause) {
+        switch (cause) {
+            case CONSENSUS_REACHED:
+                return CONSENSUS_REACHED;
+            case CONTRIBUTION_TIMEOUT:
+                return CONTRIBUTION_TIMEOUT;
```

Comment:
I find it hard to read as we have `TaskAbortCause` and `ReplicateStatusCause` with the same member names. Would it be better to statically qualify them?

Reply:
Of course :+1: